### PR TITLE
Remove OneDS and fix QoS API

### PR DESCRIPTION
**Pull request by pgilmorepf**
To allow PS4 templates to generate a working SDK again

### CODE REVIEW

File: source/test/TestApp/PlayFabEventTest.cpp [hunk 1]
```diff
@@ -169,38 +169,24 @@ namespace PlayFabUnit
...
-    void PlayFabEventTest::LightweightEvents(TestContext& testContext)
-    {
-        eventTestContext = std::make_shared<TestContext*>(&testContext);
-
-        EmitEvents(PlayFab::PlayFabEventType::Lightweight);
-    }
-
-    void PlayFabEventTest::LambdaCallbackTest(TestContext& testContext)
-    {
+    /// OneDS lightweight events (emitted individually
```
Comment (debhaldarMS) [inline source/test/TestApp/PlayFabEventTest.cpp]: >OneDS [](start = 8, length = 5)

Why are we calling these OneDS still ?

File: source/test/TestApp/PlayFabEventTest.cpp [hunk 2]
```diff
@@ -169,38 +169,24 @@ namespace PlayFabUnit
...
+    void PlayFabEventTest::OneDSEventsApi(TestContext& testContext)
```
Comment (debhaldarMS) [inline source/test/TestApp/PlayFabEventTest.cpp]: >void PlayFabEventTest::OneDSEventsApi(TestContext& testContext) [](start = 4, length = 63)

Something is screwed up with the formatting here - where are the opening braces for this method ? Unless it's not showing up in codeflow

File: source/test/TestApp/PlayFabEventTest.cpp [hunk 3]
```diff
@@ -209,9 +195,8 @@ namespace PlayFabUnit
 #endif
         AddTest("EventsApi", &PlayFabEventTest::EventsApi);
         AddTest("HeavyweightEvents", &PlayFabEventTest::HeavyweightEvents);
-        AddTest("LightweightEvents", &PlayFabEventTest::LightweightEvents);
         AddTest("LambdaCallback", &PlayFabEventTest::LambdaCallbackTest);
-        AddTest("PrivateMemberCallback", &PlayFabEventTest::PrivateMemberCallbackTest);
+        AddTest("OneDSEventsApi", &PlayFabEventTest::OneDSEventsApi);
```
Comment (debhaldarMS) [inline source/test/TestApp/PlayFabEventTest.cpp]: >OneDSEventsApi [](start = 17, length = 14)

Don't call these OneDS

File: source/test/TestApp/PlayFabEventTest.cpp (hunk 1 above)
Comment (pgilmorepf) [reply source/test/TestApp/PlayFabEventTest.cpp]: Refresh. That was a merge conflict.

File: source/test/TestApp/PlayFabEventTest.cpp (hunk 2 above)
Comment (pgilmorepf) [reply source/test/TestApp/PlayFabEventTest.cpp]: Refresh. That was a merge conflict.

File: source/test/TestApp/PlayFabEventTest.cpp (hunk 3 above)
Comment (pgilmorepf) [reply source/test/TestApp/PlayFabEventTest.cpp]: Refresh. That was a merge conflict.

**Pull request merged**